packages/component_code_gen/code_gen/generate_component_code.py

```diff
@@ -44,3 +62,17 @@ def with_docs(app, prompt, docs, docs_type, templates):
 
     logger.debug("trying again without docs")
     return no_docs(app, prompt, templates)
+
+
+def add_code_example(templates, example):
+    return templates.no_docs_system_instructions % example
```

Comment:
I'd recommend using f-strings instead of the `%` operator (again just more Pythonic, not a huge deal)